fontbe/src/orchestration.rs

```diff
@@ -352,21 +361,36 @@ impl Persistable for Marks {
     }
 }
 
-/// Precomputed kerning, to the extent possible given that we cannot create temporary var indices in advance.
-///
-/// TODO: update once <https://github.com/googlefonts/fontc/issues/571> is fixed. Then we can build a
-/// [`fea_rs::compile::PairPosBuilder`] in advance.
 #[derive(Default, Clone, Serialize, Deserialize, PartialEq)]
-pub struct Kerning {
+pub struct Kerns {
```

Comment:
hard to figure out what a "Kerns" struct is/do, how it differs from similarly named "Kerning"..
Also "kern" is more frequently used as a verb; when used as a noun, it's "a part of a typeset letter that projects beyond its side bearings" like the tails of a cursive f.  See https://www.merriam-webster.com/dictionary/kern

Admittedly I don't have an alternative to suggest at the moment.

Reply:
yea maybe something like `FinalizedKerning`, if I'm understanding the rationale correctly

Reply:
Updated to GatherIrKerning, KernFragment, and GatherKernFragments